catboost/docs/en/references/training-parameters/input.md

```diff
@@ -28,7 +28,7 @@ A comma-separated list of input files that contain the validation dataset descri
 
 {% include [files-internal-format-for-multimple-files](../../yandex_specific/_includes/reusage-formats/format-for-multimple-files.md) %}
 
-{% include [files-internal-files-internal__desc__possbile-values](../../yandex_specific/_includes/reusage-formats/files-internal__desc__possbile-values.md) %}
+{% include [files-internal-files-internal__desc__possible-values](../../yandex_specific/_includes/reusage-formats/files-internal__desc__possible-values.md) %}
```

Comment:
please rename `catboost/docs/en/yandex_specific/_includes/reusage-formats/files-internal__desc__possbile-values.md`

Reply:
As i understand files located in catboost/docs/en/yandex_specific aren't placed in GitHub. Or they are located in other branches?

Reply:
yep, yandex_specific is not exported to github
my bad, i will drop the issue

Reply:
Updated conversation message for easier internal refactor.